framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/HACepScenarioImpl.java

```diff
@@ -168,50 +170,16 @@ private File downloadAndUnzipAMQStreams() {
     }
 
     private void createTopics() {
-        final KafkaTopic masterEventsTopic = new KafkaTopicBuilder(MASTER_EVENTS_TOPIC, KAFKA_CLUSTER_NAME)
-                .withPartitions(1)
-                .withReplicas(3)
-                .addConfigItem("retention.ms", "7200000")
-                .addConfigItem("segment.bytes", "1073741824")
-                .build();
-        strimziOperator.createTopic(masterEventsTopic);
-
-        final KafkaTopic userInputTopic = new KafkaTopicBuilder(USER_INPUT_TOPIC, KAFKA_CLUSTER_NAME)
-                .withPartitions(1)
-                .withReplicas(3)
-                .addConfigItem("retention.ms", "7200000")
-                .addConfigItem("segment.bytes", "1073741824")
-                .build();
-        strimziOperator.createTopic(userInputTopic);
-
-        final KafkaTopic sessionInfosTopic = new KafkaTopicBuilder(SESSION_INFOS_TOPIC, KAFKA_CLUSTER_NAME)
-                .withPartitions(1)
-                .withReplicas(3)
-                .addConfigItem("retention.ms", "7200000")
-                .addConfigItem("segment.bytes", "1073741824")
-                .build();
-        strimziOperator.createTopic(sessionInfosTopic);
-
-        final KafkaTopic snapshotsTopic = new KafkaTopicBuilder(SNAPSHOTS_TOPIC, KAFKA_CLUSTER_NAME)
-                .withPartitions(1)
-                .withReplicas(3)
-                .addConfigItem("retention.ms", "7200000")
-                .addConfigItem("segment.bytes", "1073741824")
-                .addConfigItem("cleanup.policy", "compact")
-                .addConfigItem("segment.ms", "100")
-                .addConfigItem("min.cleanable.dirty.ratio", "0.01")
-                .addConfigItem("delete.retention.ms", "100")
-                .build();
-        strimziOperator.createTopic(snapshotsTopic);
+        final File kafkaTopicsFolder = new File(OpenShiftConstants.getHaCepSources(), SOURCES_KAFKA_TOPICS_FOLDER);
```

Comment:
If we want to use files here, it would be better to include suffix Dir to the system property for HA CEP sources.
```suggestion
        final File kafkaTopicsFolder = new File(OpenShiftConstants.getHaCepSourcesDir(), SOURCES_KAFKA_TOPICS_FOLDER);
```